src/base/utils/misc.cpp

```diff
@@ -588,24 +589,11 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath)
     // QProcess::startDetached() explorer.exe still shows the default path. In
     // this case QProcess::startDetached() probably puts its own quotes around ours.
 
-    STARTUPINFO startupInfo;
-    ::ZeroMemory(&startupInfo, sizeof(startupInfo));
-    startupInfo.cb = sizeof(startupInfo);
-
-    PROCESS_INFORMATION processInfo;
-    ::ZeroMemory(&processInfo, sizeof(processInfo));
-
-    QString cmd = QString("explorer.exe /select,\"%1\"").arg(Utils::Fs::toNativePath(absolutePath));
-    LPWSTR lpCmd = new WCHAR[cmd.size() + 1];
-    cmd.toWCharArray(lpCmd);
-    lpCmd[cmd.size()] = 0;
-
-    bool ret = ::CreateProcessW(NULL, lpCmd, NULL, NULL, FALSE, 0, NULL, NULL, &startupInfo, &processInfo);
-    delete [] lpCmd;
-
-    if (ret) {
-        ::CloseHandle(processInfo.hProcess);
-        ::CloseHandle(processInfo.hThread);
+    CoInitializeEx(NULL, COINIT_MULTITHREADED);
+    ITEMIDLIST *pidl = ILCreateFromPathW(reinterpret_cast<PCTSTR>(Utils::Fs::toNativePath(path).utf16()));
+    if (pidl) {
+        SHOpenFolderAndSelectItems(pidl, 0, nullptr, 0);
+        ILFree(pidl);
     }
```

Comment:
@glassez in case I wasn't clear, I meant using NULL for WINAPI code only. Because of shit like the one @Chocobo1 quotes:
>This parameter is reserved and must be NULL.

Theoritically old code can't distinguish between nullptr and NULL, right? If you know that WINAPI won't cause us trouble, then I have no objection using nullptr.

@Chocobo1 I doubt users will highlight files at a rate that any slowdown from initialization will be noticeable.